packages/ckeditor5-utils/src/env.js

```diff
@@ -151,3 +161,16 @@ export function isRegExpUnicodePropertySupported() {
 
 	return isSupported;
 }
+
+/**
+ * Checks if the current environment supports at least [Input Events Level 1](https://www.w3.org/TR/input-events-1/)
+ * (`input` and `beforeinput` events).
+ *
+ * @param {Window} domWindow The DOM Window interface.
+ * @returns {Boolean}
+ */
+export function areInputEventsLevel1Supported( domWindow ) {
+	const inputEvent = new domWindow.InputEvent( 'input' );
+
+	return ( 'inputType' in inputEvent ) && ( 'getTargetRanges' in inputEvent );
```

Comment:
```suggestion
    return 'inputType' in inputEvent && 'getTargetRanges' in inputEvent;
```

I just don't like redundant brackets but this might be just my personal preference 😉